pkg/api/defaults-apiserver.go

```diff
@@ -166,7 +166,7 @@ func getDefaultAdmissionControls(cs *ContainerService) (string, string) {
 	admissionControlValues := "NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,ValidatingAdmissionWebhook,ResourceQuota,ExtendedResourceToleration"
 
 	// Pod Security Policy configuration
-	if to.Bool(o.KubernetesConfig.EnablePodSecurityPolicy) {
+	if o.KubernetesConfig.IsAddonEnabled(common.PodSecurityPolicyAddonName) {
```

Comment:
validated that this apiserver defaults flow comes after the addons defaults flow (feel free to do so yourself, otherwise this would not work)